.bumpversion.cfg

```diff
@@ -12,13 +12,6 @@ serialize =
 
 [bumpversion:file:scrapy/VERSION]
 
-[bumpversion:part:prerel]
-optional_value = gamma
-values = 
-	dev
-	rc
-	gamma
-
 [bumpversion:part:prerelversion]
```

Comment:
Updated, leaving the defaults -- should be good enough for the branches.
What do you think?